app/utils/auth/JWTUtil.scala

```diff
@@ -0,0 +1,18 @@
+package utils.auth
+
+import com.mohiva.play.silhouette.api.crypto.CrypterAuthenticatorEncoder
+import com.mohiva.play.silhouette.impl.authenticators.{JWTAuthenticator, JWTAuthenticatorSettings}
+import javax.inject.Inject
+import scala.util.{Success,Failure}
+
+class JWTUtil @Inject() (
+  settings: JWTAuthenticatorSettings,
+  authenticatorEncoder: CrypterAuthenticatorEncoder) {
+
+  def unserialize(token: String): Option[JWTAuthenticator] = {
+    JWTAuthenticator.unserialize(token, authenticatorEncoder, settings) match {
```

Comment:
Are we planning to add more logic to this? If not, could we just use `.toOption`?

Also, will we use this anywhere other than `ActivateAccountController`? If not, do we need this class or can we do this inline?